docs/Manual Build Instructions.md

```diff
@@ -0,0 +1,92 @@
+## Note to the End User
+Manual builds are not recommended for end users of the Scan Engine and EtherCAT Custom Device. The build process is time consuming and requires three repositories to be cloned and built.
+
+Most end users should find the latest builds of the Scan Engine and EtherCAT Custom Device for supported versions of VeriStand at [Scan Engine and EtherCAT Custom Device Releases page](https://github.com/ni/niveristand-scan-engine-ethercat-custom-device/releases). These are automatically built, tested, and packaged into NI Packages.
+
+## Manual Build Instructions for the Scan Engine and EtherCAT Custom Device for Developers and Contributors
+Follow these steps to manually build all of the components necessary to load and use the Scan Engine and EtherCAT Custom Device.
+
+### Version Management
+All projects must be open and build in the version of LabVIEW that corresponds to the VeriStand version being targeted.
+
+For example, building for VeriStand 2020 requires using LabVIEW 2020 to build each build specification.
+
+Glossary of Terms:
+- PPL = Packed Project Library designated by the file extension .lvlibp
+- LLB = LabVIEW Library designated by the file extension .llb
+
+### 1. [Scan Engine FXP Libraries](https://github.com/ni/niveristand-scan-engine-fxp-libraries)
+1. Open `FXP LLB.lvproj`
+1. Execute `Script FXP Write.vi`
+1. Execute `Script FXP Read.vi`
+1. Build the `FXP LLB` build specification for each target
+
+The final file structure of the `Built` directory should be as follows:
+- Linux_32_ARM
+    - FXP.llb
+- Linux_x64
+    - FXP.llb
+- Pharlap
+    - FXP.llb
+
+### 2. [Scan Engine Module Libraries](https://github.com/ni/niveristand-scan-engine-module-libraries)
+1. Open `Modules.lvproj`
+1. Build the `Modules` build specification for each target
+1. Build the `NI ECAT Remote IO` build specification for each target
+
+The final file structure of the `Built` directory should be as follows:
+- Linux_32_ARM
+    - Modules.lvlibp
+    - NI ECAT Remote IO.llb
+- Linux_x64
+    - Modules.lvlibp
+    - NI ECAT Remote IO.llb
+- Pharlap
+    - Modules.lvlibp
+    - NI ECAT Remote IO.llb
+
+### 3. [Scan Engine Custom Device](https://github.com/ni/niveristand-scan-engine-ethercat-custom-device)
+
+For each target type (Linux_32_ARM, Linux_x64, and Pharlap):
+
+1. Copy dependencies built from previous repositories' `Built` directory into the `Custom Device Source` directory
+    - FXP.llb
+    - Modules.lvlibp
+    - NI ECAT Remote IO.llb
+1. Open `Scan Engine.lvproj`
+    - Depending on the versions of VeriStand and LabVIEW installed on the development PC, the version of the VeriStand .NET assemblies may mismatch with the project. Use .lvproj.config files to lock the version of the assemblies used by the project, or manually navigate to the assemblies on disk at `C:\Windows\Microsoft.NET\assembly\GAC_MSIL\`.
```

Comment:
Can we link to any existing documentation on the .lvproj.config files?

Reply:
The best (only?) documentation I could find on the subject was this old repo on NIVeriStandAdd-ons. But we'd probably need to do more documentation there before pointing people to it.
https://github.com/NIVeriStandAdd-Ons/VeriStand-LabVIEW-API-Project-Configuration-Files